src/scrabble.js

```diff
@@ -1,6 +1,34 @@
 class Scrabble {
-  // Write your implementation here
+  constructor(word) {
+    this.word = word;
+    this.onePoint = ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"];
+    this.twoPoints = ["d", "g"];
+    this.threePoints = ["b", "c", "m", "p"];
+    this.fourPoints = ["f", "h", "v", "w", "y"];
+    this.fivePoints = ["k"];
+    this.eightPoints = ["j", "x"];
+    this.tenPoints = ["q", "z"];
+  }
 
+  score() {
+    let score = 0;
+    if (typeof this.word !== "string") {
+      return score;
```

Comment:
Very concise solution. No code repetition.
Possible improvement would be not to hard code the values  score += 1, that way you could update score on variable and not have to edit code body.